include/boost/safe_float/policy/cast_from_primitive.hpp

```diff
@@ -13,39 +14,45 @@ namespace policy
 {
 namespace cast_from_primitive
 {
-template<typename FP, bool enable_other_explicitly = false, std::enable_if_t<std::is_floating_point_v<FP>, int> = 0>
+template<typename SF, bool is_explicit = false, std::enable_if_t<is_safe_float<SF>::value, int> = 0>
 struct none_impl
```

Comment:
its confusion that you have none_impl that is not meaning the same as none_implicit

Reply:
It is true, I haven't think about this confusion, I chose to mimic constructor syntax, nothing for implicit, "explicit" for explicit.
Should I rename implementation struct as name_implementation? Or put them in a implementation namespace?